modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -111,14 +111,51 @@ To delete the SSH connection data to the instance, use <<proc-deleting-connectio
 
 {payg} instance may not be accessible from the {productname} server at all times.
 
-. If the instance exists, but is stopped, the last known credentials will be used to try to connect to the instance. 
+* If the instance exists, but is stopped, the last known credentials will be used to try to connect to the instance. 
     How long the credentials remain valid depends on the cloud provider.
 
-. If the instance no longer exists, but is still registered with SUMA, its credentials are no longer valid and the authentication will fail.
+* If the instance no longer exists, but is still registered with SUMA, its credentials are no longer valid and the authentication will fail.
     The error message is shown in the column Status. 
 +
 [WARNING]
 ====
 The error message only indicates that the instance is not available. 
 Further diagnostics about the status of the instance needs to be done on the cloud provider. 
 ====
+
+
+== Registering {payg} system as a client
+
+You can register a {payg} instance from where you harvest the credentials as a {salt} client.
+The instance needs to have a valid cloud connection registered, otherwise it will not have access to channels.
+If the user removes the cloud packages, the credentials harvesting may stop working.
+
+
+Using [literal]``registercloudguest``::
+* Refreshing or changing the [literal]``registercloudguest`` connection to the public cloud update infrastructure should not interfere with the credentials usage.
+* Running [literal]```registercloudguest --clean`` can cause problems if no new cloud connection is registered with the cloud guest command.
```

Comment:
```suggestion
* Running [literal]```registercloudguest --clean`` will cause problems if no new cloud connection is registered with the cloud guest command.
```